src/frontend/sharedComponents/InvitesListener.tsx

```diff
@@ -0,0 +1,26 @@
+import {useManyInvites} from '@comapeo/core-react';
+import {useEffect} from 'react';
+import {isEditingScreen, isInviteScreen} from '../lib/screenNameChecks';
+
+export const InvitesListener = ({
```

Comment:
is there any way to make this more descriptive?

Reply:
maybe something related to handling pending invites? something like `PendingInvitesListener` would be a slight improvement at least